src/main/services/staticSite.service.ts

```
@@ -0,0 +1,502 @@
+  const visited = new Set<string>();
+  const order: SqlBlock[] = [];
+
+  function visit(name: string, chain: Set<string>) {
+    if (visited.has(name)) return;
+    if (chain.has(name)) return; // circular — skip
+    chain.add(name);
+    const block = byName.get(name);
+    if (!block) return;
+    // Find dependencies
+    const deps = [...block.sql.matchAll(/\{\{(\w+)\}\}/g)].map((m) => m[1]);
+    deps.forEach((dep) => visit(dep, new Set(chain)));
```

Biome (2.5.0)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)